### DISCUSSION TITLE
Mobile version of Fairdrive

### DISCUSSION THREAD

**Discussion by tfius** [fairDataSociety/fairdrive-theapp#421] [Ideas]
There are elements that are not needed in mobile version of Fairdrive app.
Simplest: https://cdn.osxdaily.com/wp-content/uploads/2020/01/access-edit-dropbox-files-iphone-ipad-4-369x800.jpg 
Lots of apps use ribbon/bottom selector for access to main functionality: https://socialmedianz.com/wp-content/uploads/2013/04/photo-6.png 
or 
(https://aem.dropbox.com/cms/content/dam/dropbox/blog/files/2015/05/ios-screenshots_final.png) 

I propose 
1. left toolbar with "Overview, Drive, Explore", to be hidden, 
2. add bottom toolbar: Private, Shared, Create Pod,
3. Left side would only contain list of pods

**Comment by tfius** [comment]
wdyt @tomicvladan @IgorShadurin @tamas6 ?

**Reply by tamas6** [reply]
missing_searchbar_ option is the real problem if you ask me, but there is no aggragation of data as I know, so we could at least list those last used items, that would be a dealbreaker imo

**Reply by tfius** [reply]
i agree, wonder how much would a proper search implementation require

**Comment by IgorShadurin** [comment]
Yes, it's a good idea for a mobile version.
Regarding the left side: we can remove it completely. Pods can be displayed on the first screen.
However, the logic of all mobile UI screens should be thought through in a complex. In order not to get Frankenstein with the current version.
